Makefile

```diff
@@ -17,6 +19,12 @@ export CRI_DOCKERD_LDFLAGS=-ldflags "-s -w -buildid=${REVISION} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.PreRelease=${PRERELEASE} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.GitCommit=${REVISION}"
 
+all: cri-dockerd
+
+.PHONY: cri-dockerd
+cri-dockerd:
+	GOARCH=$(ARCH) go build -trimpath $(CRI_DOCKERD_LDFLAGS) -o $@
```

Comment:
There is no "bin" folder in the binary folder or in the default go build command, so it would have to be added explicitly

Reply:
I was thinking there were other command using bin and it's nice to keep build artifacts cleaned up.

I'm actually seeing now that `make static-linux` does the same as this command and outputs the binary to packaging/static/build/linux/cri-dockerd. We should use that command and point users to that folder rather than duplicating it.

Reply:
I think the main difference is that it builds a static binary ?

Reply:
Anyway, the main difference from the previous install was that it didn't (re)build the binary again.

```diff
 cd cri-dockerd
-mkdir bin
-go build -o bin/cri-dockerd
 mkdir -p /usr/local/bin
-install -o root -g root -m 0755 bin/cri-dockerd /usr/local/bin/cri-dockerd
+install -o root -g root -m 0755 cri-dockerd /usr/local/bin/cri-dockerd
 install packaging/systemd/* /etc/systemd/system
```

So that the user could choose to either do `make cri-dockerd`, or to download it from "Releases"

Reply:
Yep. It looks like it has some debugging information stripped. I'm fine with the new one being considered a "dev" version